src/openforms/registrations/contrib/zgw_apis/typing.py

```diff
@@ -41,6 +41,8 @@ class RegistrationOptions(TypedDict):
     organisatie_rsin: NotRequired[str]
     zaak_vertrouwelijkheidaanduiding: NotRequired[VertrouwelijkheidAanduiding]
     medewerker_roltype: NotRequired[str]
+    partners_roltype: NotRequired[str]
+    partners_description: NotRequired[str]
```

Comment:
can you put proper defaults on the serializer fields so that we can omit the `NotRequired` here? :) it makes handling the options in the client/plugin code simpler